internal/cmd/skupper/utils/handle_error.go

```diff
@@ -18,7 +18,7 @@ func HandleErrorList(errList []error) {
 			fmt.Println(err)
 		}
 
-		syscall.Exit(0)
+		syscall.Exit(2)
```

Comment:
What motivated the use of 2 as the exit code here? Not saying it is wrong, just curious.

Reply:
It was based on something said in this article: https://www.redhat.com/sysadmin/exit-codes-demystified

> Exit status 2 appears when there's a permissions problem or a missing keyword in a command or script.

I thought it was kind of related, because the `HandleErrorList` displays all the validation errors found in the command execution.

Reply:
Possibly adding a list of constants to help identifying the purposes of the error codes, might help.

Reply:
Aggreed :+1: